avidemux/qt4/i18n/avidemux_da.ts

```diff
@@ -10376,7 +9352,7 @@ Hvis det er en systemprofil, så vil den blive genskabt næste gang.</translatio
         <location line="+179"/>
         <location line="+302"/>
         <source>%</source>
-        <translation type="unfinished"></translation>
+        <translation>%</translation>
```

Comment:
Don't add a translation which doesn't alter the source. This applies to a lot of entries below.